### DISCUSSION TITLE
How to manager URL (SEO) of page

### DISCUSSION THREAD

**Discussion by thienvc** [dnnsoftware/Dnn.Platform#4843] [Q&A]
![image](https://user-images.githubusercontent.com/1115393/135188461-1b726537-4e5d-4111-8259-3b993910e04a.png)

Some are allowed to be deleted, others are not allowed
Only one is allowed to edit...
How to specify the one primary URL?

**Comment by sleupold** [comment]
you may edit only manually created URLs, the automatic ones are created by the system according to the menu position (not deletable) or after renaming a page.

**Comment by valadas** [comment]
One improvement I would like to make in this area is to allow deleting system ones as long as it's not the last 200 one. I _think_ there might be an issue already logged for that.
